lib/shared/types/src/types/apis/sdk/serverSDKTypes.ts

```diff
@@ -0,0 +1,86 @@
+import { DVCLogger, DVCDefaultLogLevel } from '../../../logger'
+import { DVCReporter } from '../../../reporter'
+
+/**
+ * Options to control the setup of the DevCycle NodeJS Server SDK.
+ */
+export interface DevCycleOptions {
```

Comment:
DevCycleServerSDKOptions?

Reply:
I think this is re-exported as a public interface for the SDKs so would likely need to be renamed there...